go.mod

```diff
@@ -10,17 +10,17 @@ require (
 	github.com/fluxcd/pkg/helmtestserver v0.0.1
 	github.com/fluxcd/pkg/lockedfile v0.0.5
 	github.com/fluxcd/pkg/recorder v0.0.6
-	github.com/fluxcd/pkg/runtime v0.0.1
+	github.com/fluxcd/pkg/runtime v0.0.3
 	github.com/fluxcd/pkg/ssh v0.0.5
 	github.com/fluxcd/pkg/untar v0.0.5
-	github.com/fluxcd/source-controller/api v0.0.16
+	github.com/fluxcd/source-controller/api v0.0.17
 	github.com/go-git/go-billy/v5 v5.0.0
 	github.com/go-git/go-git/v5 v5.1.0
 	github.com/go-logr/logr v0.1.0
 	github.com/minio/minio-go/v7 v7.0.5
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
-	helm.sh/helm/v3 v3.3.1
+	helm.sh/helm/v3 v3.3.2
```

Comment:
I think this change should be put into the changelog.

Reply:
Done!